map/bookmark_manager.cpp

```diff
@@ -1867,7 +1867,7 @@ void BookmarkManager::LoadBookmarkRoutine(std::string const & filePath, bool isT
     std::string fileSavePath = GetKMLPath(filePath);
     if (!fileSavePath.empty())
     {
-      auto const ext = GetLowercaseFileExt(filePath);
+      auto const ext = GetLowercaseFileExt(fileSavePath);
```

Comment:
When you import KMB file it is converted to KML first in `GetKMLPath(...)` function and then KML is imported.
In such case `filePath = "some_file.kmb` and `fileSavePath = "some_file.kmb2.kml"`. Because we parse `fileSavePath` we should get extension correctly here.

Reply:
It would be even better to see these comments in the code instead of the PR ;-)

Reply:
Reviewed this part of code because `ext` is always `"kml"`.

Reply:
Did you add a comment in the code? Btw, why is it always kml there?